docs/source/recipes/images.md

```diff
@@ -0,0 +1,66 @@
+---
+myst:
+  html_meta:
+    "description": "Volto code recipe for images"
+    "property=og:description": "Volto code recipe for images"
+    "property=og:title": "Volto code recipe for images"
+    "keywords": "Plone, Volto, code, recipe, images"
+---
+
+(images-label)=
+
+# Images
+
+Volto comes with an `Image` component that loads images in an optimal way.
+
+Features of the `Image` component are the following.
+
+- optimized [lazy loading](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading)
+- automatic `srcset` generation, if given a content item or catalog brain
+- optional responsive width
+
+The component can accept all the regular HTML `<img>` attributes and a few extra for the above features.
```

Comment:
````suggestion
The component can accept all the regular HTML `<img>` attributes and a few extra for the above features.
The following example demonstrates how the code with attributes will render to HTML.

```jsx
import config from '@plone/volto/registry';

const Image = config.getComponent({ name: 'Image' }).component;

<Image
  item={content}
  imageField="image"
  alt="Alt text for accessibility"
  class="myClass"
/>;
```

```html
<img src="my-image.png" alt="Alt text for accessibility" class="myClass" />
```
````

Reply:
@stevepiercy should be className, not class

Reply:
And I think we need either this example or the one below, but not both.

Reply:
@davisagli thanks, corrected. I would prefer the above complete example, too. It's the first one and illustrates the main concepts.

I think the (now) second example is still useful, but may have its `import` and `const` removed for brevity.

Reply:
What about using only basic HTML attributes here and an item `content` in the example below to show specifically the extra feature of using the content object?

This first example would become

```jsx
import config from '@plone/volto/registry';

const Image = config.getComponent({ name: 'Image' }).component;

<Image
  src="https://picsum.photos/200/300"
  alt="Alt text for accessibility"
  className="myClass"
  width="200"
  height="300"
/>
```

```html
<img
  src="https://picsum.photos/200/300"
  alt="Alt text for accessibility"
  class="myClass"
  width="200"
  height="300"
/>
```

Reply:
That works for me. @davisagli?

Reply:
Sure!

Reply:
Done